webhooks.md

```diff
@@ -0,0 +1,105 @@
+# Webhooks
+
+Partners can be notified about integration changes via webhook calls that are triggered everytime when hotel updates the integration.
```

Comment:
Honza's suggestion: `hen an action of particular type is performed in the system, a message is sent to the configured webhook, using HTTP POST and JSON body.`